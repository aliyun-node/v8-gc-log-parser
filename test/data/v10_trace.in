[25830:0x4523210]    30656 ms: Scavenge 592.8 (689.7) -> 579.8 (690.2) MB, 2.6 / 0.0 ms  (average mu = 0.981, current mu = 0.986) allocation failure 
[25830:0x4523210]    30694 ms: Scavenge 594.2 (690.2) -> 581.2 (691.2) MB, 3.0 / 0.0 ms  (average mu = 0.981, current mu = 0.986) allocation failure 
[25830:0x4523210]    30743 ms: Scavenge 595.4 (691.2) -> 582.4 (691.7) MB, 3.0 / 0.0 ms  (average mu = 0.981, current mu = 0.986) allocation failure 
