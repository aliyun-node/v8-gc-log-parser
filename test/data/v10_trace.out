[
  {
    "pid": 25830,
    "isolate": "0x4523210",
    "time_since_init": 30656,
    "data": {
      "collector": "scavenge",
      "start_object_size": 592.8,
      "end_object_size": 579.8,
      "start_memory_size": 689.7,
      "end_memory_size": 690.2,
      "pause": 2.6,
      "external_time": 0,
      "step_description": "average mu = 0.981, current mu = 0.986"
    },
    "type": "trace"
  },
  {
    "pid": 25830,
    "isolate": "0x4523210",
    "time_since_init": 30694,
    "data": {
      "collector": "scavenge",
      "start_object_size": 594.2,
      "end_object_size": 581.2,
      "start_memory_size": 690.2,
      "end_memory_size": 691.2,
      "pause": 3,
      "external_time": 0,
      "step_description": "average mu = 0.981, current mu = 0.986"
    },
    "type": "trace"
  },
  {
    "pid": 25830,
    "isolate": "0x4523210",
    "time_since_init": 30743,
    "data": {
      "collector": "scavenge",
      "start_object_size": 595.4,
      "end_object_size": 582.4,
      "start_memory_size": 691.2,
      "end_memory_size": 691.7,
      "pause": 3,
      "external_time": 0,
      "step_description": "average mu = 0.981, current mu = 0.986"
    },
    "type": "trace"
  }
]
